src/utils/db.js

```
@@ -0,0 +1,9 @@
+import { Sequelize } from 'sequelize';
+
+export const client = new Sequelize({
+  host: process.env.DB_HOST,
+  username: process.env.DB_USER,
+  password: process.env.DB_PASSWORD,
+  database: process.env.DB_DATABASE,
```

If required env vars (DB_HOST, DB_USER, DB_PASSWORD, DB_DATABASE) are missing the client will fail later with a connection error. Consider validating these env vars at startup and failing fast with a clear error message or providing development defaults to help debugging.